WORKSPACE

```diff
@@ -29,3 +29,17 @@ maven_jar(
   sha1 = "e5b3e2753d0817b622c32aedcb888bcf39e275b4")
 
 
+# dependencies needed for BazelTestRunner
+maven_server(
+    name = "one_hippo",
+    url = "http://maven.onehippo.com/maven2/",
+)
+
+maven_jar(
```

Comment:
@johnynek This should probably move to the *_repositories pattern and bind, right?